lib/iris/fileformats/pp.py

```diff
@@ -2028,6 +2028,48 @@ def load_cubes(filenames, callback=None, constraints=None):
                                        constraints=constraints)
 
 
+def as_cubes(pp_fields, callback=None):
+    """
+    Convert an iterable of PP fields into an iterable of Cubes.
+
+    Args:
+
+    * pp_fields:
+        An iterable of :class:`iris.fileformats.pp.PPField`.
+
+    * callback - a function which can be passed on to
```

Comment:
With the caveat that you can't actually do anything with the filename argument, but it still has to be there?